__tests__/flavored-compilers/__snapshots__/mermaid.test.js.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Compile Mermaid blocks compiles a mermaid block 1`] = `
+"\`\`\`mermaid-render-svg
+graph LR
+    A[Will it blend?] -> B[Yes]
```

Comment:
👏

Reply:
Oops, that's not a valid graph, should be `-->`